python/ray/tune/ray_trial_executor.py

```diff
@@ -608,7 +617,8 @@ def restore(self, trial, checkpoint=None):
                 trial.sync_logger_to_new_location(worker_ip)
             with warn_if_slow("restore_from_disk"):
                 # TODO(ujvl): Take blocking restores out of the control loop.
-                ray.get(trial.runner.restore.remote(value))
+                with self._switch_working_directory(trial):
```

Comment:
can you swap this with the above `warn_if_slow` on L618? This is just so that we are consistent with making sure the timing is within the context.

Reply:
Done for all similar cases 7315f75